solarathon/pages/dashboard.py

```diff
@@ -48,13 +67,40 @@ def format_price(price):
     formatted_price = '{:,.0f}'.format(float(price)).replace(',', ' ')
     return formatted_price
 
+
+def get_binance_ticket(symbol: str) -> TickerData:
+    binance_url = f"https://api.binance.com/api/v3/ticker/24hr?symbol={symbol}"
+    return TickerData.model_validate_json(
+        json_data=requests.get(binance_url).content
+    )
+
+
 @solara.component
-def DashboardCard(price: str, procentChange: str, icon: solara.Element = None, market_cap: Optional[str] = None,  market_cap_change_percentage: Optional[str] = None, pending: Optional[bool] = False): 
-    if pending:
-             with solara.Card(
-            GeckoIcon('', '')
-            , style={"width":"330px", "min-width": "280px","max-width": "350px", "background-color": "#1B2028", "color": "#ffff", "border-radius": "16px", "padding": "20px", "box-shadow": "rgba(0, 0, 0, 0) 0px 0px, rgba(0, 0, 0, 0) 0px 0px, rgba(0, 0, 0, 0.2) 0px 4px 6px -1px, rgba(0, 0, 0, 0.14) 0px 2px 4px -1px"}, margin=0, classes=["my-2", "mx-auto",]):
-    
+def DashboardCard(symbol: str, icon: solara.Element | str = None, market_cap: Optional[str] = None,  market_cap_change_percentage: Optional[str] = None, pending: Optional[bool] = False):
```

Comment:
for union type it tells me `TypeError: unsupported operand type(s) for |: 'type' and 'type'`